POLYFILLS.md

```diff
@@ -1,21 +1,26 @@
 # Polyfills
 
-`<model-viewer>` relies on standardized features of the Web Platform to do a lot of
-what it does. Some of these features are very new and don't exist in all
+`<model-viewer>` relies on standardized features of the Web Platform to do a lot
+of what it does. Some of these features are very new and don't exist in all
 browsers yet. In order to maximize browser compatibility, you should install
 [polyfills](https://en.wikipedia.org/wiki/Polyfill_(programming)) to fill in the
 gaps for some of the newest features.
 
 ## Browser Support
 
-The following emerging standard web platform APIs are needed by this library:
+The following emerging standard web platform APIs are required by
+`<model-viewer>`:
 
  - [Custom Elements](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements) ([CanIUse](https://caniuse.com/#feat=custom-elementsv1), [Platform Status](https://www.chromestatus.com/features/4696261944934400))
  - [Shadow DOM](https://dom.spec.whatwg.org/#shadow-trees) ([CanIUse](https://caniuse.com/#feat=shadowdomv1), [Platform Status](https://www.chromestatus.com/features/4667415417847808))
- - [Resize Observer](https://wicg.github.io/ResizeObserver/) ([CanIUse](https://caniuse.com/#feat=resizeobserver), [Platform Status](https://www.chromestatus.com/features/5705346022637568))
  - [Intersection Observer](https://w3c.github.io/IntersectionObserver/) ([CanIUse](https://caniuse.com/#feat=intersectionobserver), [Platform Status](https://www.chromestatus.com/features/5695342691483648))
  - [Fullscreen API](https://fullscreen.spec.whatwg.org/) ([CanIUse](https://caniuse.com/#feat=fullscreen), [Platform Status](https://www.chromestatus.com/features/6596356319739904))
 
+The following emerging web platform API is optional, and will be used by
+`<model-viewer>` if it is detected on the page:
```

Comment:
It's unclear what happens if I do not have ResizeObserver support, what feature I miss out on, or cost I incur, or info to weigh the decision of "should I include the polyfill"

Reply:
👍 will get more details in